Netro Sprinklers.indigoPlugin/Contents/Server Plugin/api_client.py

```
@@ -54,8 +55,18 @@
+class DeviceTokenState:
+    """Per-device API token budget tracking.
+
+    Each Netro device has its own independent 2000/day token limit.
+    """
+    token_remaining: int = 2000
+    token_reset: Optional[datetime] = None
+
+
 # Re-export threshold constants for convenient access
-__all__ = ["NetroAPIClient", "TOKEN_PAUSE_THRESHOLD", "TOKEN_WARNING_THRESHOLD"]
+__all__ = ["NetroAPIClient", "DeviceTokenState", "TOKEN_PAUSE_THRESHOLD", "TOKEN_WARNING_THRESHOLD"]
```

Ruff (0.15.9)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)